### PR TITLE
ruben [ENG-268]: feat: telegram limits command

### PR DESCRIPTION
**Pull request by topether21**
How to test:

- npm i
- npm run start
- go to telegram
- add /limit command

- npm run hunter:limits 

https://github.com/deezy-inc/sat-hunter/assets/126987350/eee0b001-7800-45ff-87ba-de67091f909d

image

### CODE REVIEW

File: __tests__/commands.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+        expect(result).toEqual({
+            payment_details: `
+Your current limits and payment info:
+
+BTC Volume Permitted Every 30 Days: 100.
+Subscription Cost: 200.
+Cost to purchase 1 additional BTC in scan volume: 300 satoshis.
+You have scanned 400 BTC so far this billing period.
+Payment Address:
```
Comment (habibitcoin) [inline __tests__/commands.js:28]: Your current limits and payment info:
Volume Permitted Every 30 Days: 400 BTC
Subscription Cost: 0.4 BTC 
Cost to purchase 1 additional BTC in scan volume: 300 satoshis.
You have scanned 111.86624825 BTC so far this billing period.
Payment Address:

File: notifications.js [hunk 2]
```diff
@@ -1,61 +1,28 @@
...
-        success = true
-      } catch (err) {
-        console.log(err)
-        retries++
-      }
-    }
+const initNotifications = async () => {
+  if (TELEGRAM_BOT_ENABLED) {
+    console.log(`Telegram bot is enabled`)
+    await initCommands()
```
Comment (habibitcoin) [inline notifications.js:20]: Undefined error

**Comment by dannydeezy** [comment]
@topether21 why was `setup-telegram-bot.js` deleted? i think we still need that to set up the bot in the first place

**Comment by dannydeezy** [comment]
I think the response should look more like this (less words, less info)
```
Scan Credits Remaining: 3000 BTC

Payment Address: bc1p.....

```

I'm having a hard time understanding what all the other values mean. Especially they are confusing now that some people are on unlimited monthly plans. @habibitcoin maybe we should chat about this?

**Review by habibitcoin — APPROVED** [review]
Made a few changes to address comments. Good to go now!

https://github.com/deezy-inc/sat-hunter/pull/32/commits/c852b10c665ecbb2b45553c47788d38651e20329

Also in your next sat-hunter PR, could you please refactor the tests in /tests into _tests_ and remove /tests @topether21 ?

**Comment by dannydeezy** [comment]
Getting this error when running:
```
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]: /home/ubuntu/sat-hunter/notifications.js:19
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]:     await initCommands()
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]:     ^
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]: ReferenceError: initCommands is not defined
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]:     at initNotifications (/home/ubuntu/sat-hunter/notifications.js:19:5)
Dec 12 17:23:59 ip-172-31-20-222 node[3846472]:     at runLoop (/home/ubuntu/sat-hunter/index.js:310:11)
```

File: __tests__/commands.js (hunk 1 above)
Comment (dannydeezy) [reply __tests__/commands.js:28]: I would remove the "Your current limits and payment info:" line entirely, its unnecessary

Comment (dannydeezy) [reply __tests__/commands.js:28]: maybe like this:?
```
Scan Volume Remaining: 2300 BTC

Price: 1000 sats / BTC

Payment Address: bc1p....
```

Comment (dannydeezy) [reply __tests__/commands.js:28]: per convo with @habibitcoin 
```
{ if !unlimited }
Scan Volume Remaining: 2300 BTC
Price: 1000 sats / BTC
{ end }
{ if !free_tier }
Tier: X BTC per Y days
Subscription Cost: 0 BTC
[ end }
Payment Address:
bc1pwk6
```

**Comment by habibitcoin** [comment]
Going to update text to:
```
{ if !unlimited }
Scan Volume Remaining: 2300 BTC
Price: 1000 sats / BTC
{ end }
{ if !free_tier }
Tier: X BTC per Y days (if x < 0, unlimited)
Subscription Cost: 0 BTC
[ end }
Payment Address:
bc1pwk6z3fje3jpk4jd3qyy0q8qace49exquuy3epqkxkk6488v8rh4qvr0kg8
```

Also need to fix npm run setup-telegram

**Comment by habibitcoin** [comment]
1 more note: make sure it keeps running at all times

**Pull request merged**